wp-content/mu-plugins/pantheon/pantheon-plugin-install-notice.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * If a Pantheon site is in Git mode, hide the Plugin installation functionality and show a notice.
+ */
+
+if ( ! wp_is_writable( WP_PLUGIN_DIR ) ) {
+    if ( ! defined( 'DISALLOW_FILE_MODS' ) ) {
+        define( 'DISALLOW_FILE_MODS', true );
+    }
+
+    add_action( 'admin_notices', '_pantheon_plugin_install_notice' );
+    add_action( 'network_admin_notices', '_pantheon_plugin_install_notice' );
+}
+
+function _pantheon_plugin_install_notice() {
+    $screen = get_current_screen(); 
+    // Only show this notice on the plugins page.
+    if ( 'plugins' === $screen->id || 'plugins-network' === $screen->id ) { ?>
+        <div class="update-nag notice notice-warning is-dismissible" style="margin: 5px 6em 15px 0;">
+            <p style="font-size: 14px; margin: 0;">
+                If you wish to update or add plugins using the WordPress UI, switch your site to SFTP mode from <a href="https://dashboard.pantheon.io/sites/<?php echo $_ENV['PANTHEON_SITE']; ?>">your Pantheon dashboard</a>.
```

Comment:
👋  Drive-by CR comment 🚗 💨 
You might consider i18nizing the notice. I realize basically nothing else in the Pantheon mu-plugin is internationalized but it will be marginally easier to correct that (which I see as an inevitability as Pantheon becomes more global/international) if we start addressing that incrementally as we're making changes/improvements rather than having to do it all in one big go.